modules/ROOT/pages/provisioning-kubevirt.adoc

```diff
@@ -0,0 +1,96 @@
+= Provisioning Fedora CoreOS on KubeVirt
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on any KubeVirt-enabled Kubernetes cluster.
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a Kubernetes environment with https://kubevirt.io/user-guide/operations/installation/[KubeVirt] installed.
+
+== Downloading the KubeVirt Image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+Once you have picked the relevant stream, download the ociarchive:
+
+NOTE: For more information on FCOS stream offerings see xref:update-streams.adoc[Update Streams].
+
+[source, bash]
+----
+STREAM='stable'
+coreos-installer download -s $STREAM -p kubevirt -f ociarchive
+----
+
+Alternatively, you can manually download a KubeVirt image from the
+https://getfedora.org/coreos/download?tab=cloud_operators[download page] and directly download the ociarchive from there.
+
+== Uploading the Image to a Container Registry
+
+.Upload the image to your container registry
```

Comment:
Do we not plan to upload the image ourselves as part of the release process?

Reply:
Added a comment there.  If possible, let's get a push location worked out and avoid the copy awkwardness.  We should point users directly to the stream tags in our repo, though, rather than having them copy the image elsewhere.  We might still want to document the download/upload flow, in case there are users using private registries without direct Internet access, but let's put it at the bottom of the page and document it as an advanced case.

Reply:
:+1: Updated with a possible draft on how this would look like.